components/navbar.css

```diff
@@ -0,0 +1,146 @@
+.logo {
+    flex-basis: 10%;
+    margin-right: 5rem;
+}
+
+.user i {
+    -webkit-text-stroke: 1px var(--font-color-dark);
+}
+
+i {
+    cursor: pointer;
+}
+
+.right-nav {
+    align-items: center;
+    flex-basis: 50%;
+}
+
+.form-search {
+    border: 2px solid var(--color-dark-blue);
+    border-radius: var(--size-12);
+    box-shadow: var(--box-shadow-2);
+    width: 100%;
+    min-width: 2rem;
+}
+
+#search-field {
+    border: transparent;
+    outline: transparent;
+}
+
+.text-rg {
+    margin: 0;
+}
+
+.search-icon {
+    color: var(--accent-color);
+}
+
+header {
+    position: sticky;
+    top: 0;
+    right: 0;
+    left: 0;
+    box-shadow: var(--box-shadow);
+    background-color: var(--background-color);
+    z-index: var(--z-1);
+}
+
+/* Account Options */
+
+.dropbtn {
+    border: none;
+    cursor: pointer;
+}
+
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: var(--light-grey);
+    min-width: 160px;
+    overflow: auto;
+    z-index: 1;
```

Comment:
If you have a defined variable for `z-index` then use it here.

Reply:
Yes I have. Would make the change.